php/libraries/User.class.inc

```diff
@@ -252,6 +252,24 @@ class User extends UserPermissions
     }
 
     /**
+     * Returns all user's sites that are StudySites
+     *
+     * @return array
+     */
+    function getStudySites()
+    {
+        $site_arr         = $this->userInfo['CenterIDs'];
+        $user_study_sites = array();
+        foreach ($site_arr as $key => $val) {
+            $site[$key] = &Site::singleton($val);
```

Comment:
@driusan @ridz1208 

is it acceptable to call the Site class from within the User class? or is it better in this function to do a database query instead without relying on a call to another class?